cobc/tree.h

```diff
@@ -1590,7 +1590,94 @@ extern void		ppparse_clear_vars (const struct cb_define_struct *);
 extern struct cb_define_struct *ppp_search_lists (const char *name);
 
 extern void		plex_action_directive (const enum cb_directive_action,
-	const unsigned int);
+					       const unsigned int);
+
+/* COPY/REPLACING/REPLACE */
+
+/* COPY extended syntax defines */
+enum cb_replace {
+	CB_REPLACE_ALL		= 0,
+	CB_REPLACE_LEADING	= 1,
+	CB_REPLACE_TRAILING	= 2,
+};
+
+/* Strict/loose source text replacement structure */
+struct cb_replace_src {
+	const struct cb_text_list	*text_list; /* single-cell when strict */
+	enum cb_replace			lead_trail;
+	unsigned int			strict: 1;
```

Comment:
Is there any use in packing a single attribute?

Reply:
Just declaring that a single bit is used, in case other flags need to be added later on. So let's say this acts like a comment.